test/integration/targets/nxos_aaa_server_host/tests/common/tacacs.yaml

```diff
@@ -133,6 +133,23 @@
 
   - assert: *false
 
+  - name: "Configure tacacs server with new type 7 encryption key"
+    nxos_aaa_server_host:
+      server_type: tacacs
+      address: 8.8.8.8
+      host_timeout: 25
+      tacacs_port: 89
+      encrypt_type: 7
+      key: helloback
+      provider: "{{ connection }}"
+      state: present
+    register: result
+
+  - assert:
+      that:
+        - "result.changed == true"
+        - "'key 7' in result.updates[0]"
```

Comment:
Please add idempotence test.

Reply:
Hi Trishna, I've added the idempotence tests. Cheers, Dan